impeller/renderer/render_target.h

```diff
@@ -18,32 +18,21 @@ namespace impeller {
 
 class Context;
 
-/// @brief a wrapper around the impeller [Allocator] instance that can be used
-///        to provide caching of allocated render target textures.
-class RenderTargetAllocator {
- public:
-  explicit RenderTargetAllocator(std::shared_ptr<Allocator> allocator);
-
-  virtual ~RenderTargetAllocator() = default;
-
-  /// @brief Create a new render target texture, or recycle a previously
-  /// allocated render
-  ///        target texture.
-  virtual std::shared_ptr<Texture> CreateTexture(const TextureDescriptor& desc);
-
-  /// @brief Mark the beginning of a frame workload.
-  ///
-  ///       This may be used to reset any tracking state on whether or not a
-  ///       particular texture instance is still in use.
-  virtual void Start();
-
-  /// @brief Mark the end of a frame workload.
-  ///
-  ///        This may be used to deallocate any unused textures.
-  virtual void End();
-
- private:
-  std::shared_ptr<Allocator> allocator_;
+struct RenderTargetConfig {
```

Comment:
This is a subset of the render target properties that we care about for caching (at the entity level).

Reply:
Its defined here for ... reasons, but I could move this up to the entity cache so its more obvious